torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -3190,4 +3190,25 @@ at::Tensor XLANativeFunctions::_cdist_forward(
       bridge::GetXlaTensor(x1), bridge::GetXlaTensor(x2), p));
 }
 
+std::tuple<at::Tensor, at::Tensor, at::Tensor> XLANativeFunctions::_unique2(
```

Comment:
FYI: I don't think this PR will be impacted but just in case you need to develop dynamic models, you'd need to develop on `functionization_dynamic_shape` branch which has functionalization feature enabled.